tests/lists.test.ts

```diff
@@ -0,0 +1,551 @@
+import * as mocha from "mocha";
```

Comment:
One of the things we talked about in class is making sure there are some tests that are inside functions – can you please add some tests that have lists as function arguments, or create lists inside of functions?

You actually have a lot of great tests, but these would help make sure a number of things regarding scope and list creation work well.

Reply:
We've added new test cases where
1) lists are used as function parameters
2) lists are returned from functions